packages/apps/src/microsoft/teams/apps/token_manager.py

```diff
@@ -70,29 +51,53 @@ async def get_graph_token(self, tenant_id: Optional[str] = None, force: bool = F
         Returns:
             The graph token or None if not available
         """
-        if not self._credentials:
-            self._logger.debug("No credentials provided for graph token refresh")
+        return await self._get_token("https://graph.microsoft.com/.default", tenant_id)
+
+    async def _get_token(
+        self, scope: str | list[str], tenant_id: str | None = None, *, caller_name: str | None = None
```

Comment:
always wondered about this, but is it better to sandwich the * ?

Reply:
Hm, not sure tbh. I just think of it as stuff that's essential for the operation of a function vs not. I'll actually remove the default param from tenant_id